.github/workflows/gh-sync.yml

```diff
@@ -0,0 +1,34 @@
+name: Sync GitHub with ADO
+
+on: 
+  issues:
+    types: [closed, edited, deleted, reopened, assigned, unassigned, labeled, unlabeled]
+  issue_comment:
+
+jobs:
+  sync-issues:
+    name: Run gh-sync from GitHub action
+    if: ${{ github.event.label.name == 'tracking' || contains(github.event.issue.labels.*.name, 'tracking') }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Azure
+        uses: Azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - id: AzureKeyVault
+        uses: Azure/get-keyvault-secrets@v1
+        with:
+          keyvault: 'kv-qdk-build'
+          secrets: 'ghSyncBuildPAT'
+
+      - name: 'Trigger gh-sync'
+        uses: microsoft/gh-sync@main
+        with:
+          ado-organization-url: 'https://ms-quantum.visualstudio.com/'
```

Comment:
Ah, that's a good point, agreed. Adding to `{{ secrets.* }}` would allow for changing values without a PR, and for reserving internal implementation details such as project URLs and area paths.

Reply:
Good note, I've updated each of the ADO variables to be referenced from environment variables.

Reply:
LGTM! Happy for this to be merged in whenever you're ready, then.